packaging/deb/debian/source/include-binaries

```diff
@@ -0,0 +1,47 @@
+debian/mackerel-plugin-apache2
```

Comment:
It seems necessary. So, it will be generated automatically by the release script.